orchagent/port/portcnt.h

```diff
@@ -172,6 +172,40 @@ class PortConfig final
         bool is_set = false;
     } description; // Port description
 
+    struct {
+        sai_link_event_damping_algorithm_t value;
+        bool is_set = false;
+    } link_event_damping_algorithm; // Port link event damping algorithm
+
+    struct {
+
+        struct {
+            uint32_t value;
```

Comment:
is it possible to use std::optional or boost::optional here?

Reply:
Hi @Junchao-Mellanox, thanks for your review. There was a refactor change in portsorch.cpp, and my change follows the existing port config. Maybe there can be a cleanup task in the future to update the port config to use std::optional.